packages/l/libhybris/xmake.lua

```diff
@@ -0,0 +1,37 @@
+package("libhybris")
+    set_homepage("https://github.com/libhybris/libhybris")
+    set_description("Hybris is a solution that commits hybris, by allowing us to use bionic-based HW adaptations in glibc systems ")
+
+    add_urls("https://github.com/libhybris/libhybris.git")
+    add_versions("2024.09.01", "936279916605003fba95a0f3629a6bc5e6caa343")
+
+    add_deps("autoconf", "automake", "libtool", "pkgconf", "ndk")
+
+    on_install("linux", function (package)
```

Comment:
if you are using ndk to build android program, please use `android` platform, not linux

Reply:
This package is similar to dyncall and allows linux to use android .so files